packages/schematics/angular/migrations/update-6/index.ts

```diff
@@ -268,10 +315,7 @@ function extractProjectsConfig(config: CliConfig): JsonObject {
         }
       testOptions.scripts = (app.scripts || []).map(extraEntryMapper);
```

Comment:
Scripts and styles mapping must use `bundleName` instead of `output`. For lazy ones, the bundle name must be determined manually by removing the file extension of the input.

Reply:
Turns out that we didn't have an `output` for scripts and styles.